scripts/docker-cleanup.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e -u
+
+echo "Cleaning APT"
+sudo apt-get clean
+sudo rm -rf /var/lib/apt/lists/*
+
+echo "Cleaning unneeded NDK modules"
+cd /home/builder/lib/android-ndk/sources
+rm -Rf cxx-stl/system third_party/shaderc third_party/vulkan
+cd /home/builder/lib/android-ndk/platforms
+rm -Rf android-16 android-17 android-18 android-19
+cd /home/builder/lib/android-ndk
+rm -Rf shader-tools
+
+echo "Cleaning unneeded SDK modules"
+cd /home/builder/lib/android-sdk/tools
+mkdir -p lib2
+cp $(grep "CLASSPATH=" bin/sdkmanager | head -n 1 | cut -d '=' -f 2 | tr ':' ' ' | sed 's%\$APP_HOME/%%g') lib2
+rm -Rf emulator* lib proguard template support/*.txt
+mv lib2 lib
+cd /home/builder/lib/android-sdk/platforms/android-21
+rm -Rf data templates skins
+cd /home/builder/lib/android-sdk/platforms/android-28
+rm -Rf data templates skins
+
+echo "Zipping notices"
+cd /home/builder/lib/android-ndk
+bzip2 NOTICE NOTICE.toolchain sysroot/NOTICE
+cd /home/builder/lib/android-sdk
+bzip2 tools/NOTICE.txt build-tools/28.0.3/NOTICE.txt platform-tools/NOTICE.txt
```

Comment:
Could you source `scripts/properties.sh` and use the TERMUX_ANDROID_BUILD_TOOLS_VERSION variable defined there to avoid hardcoding 28.0.3 here?

Reply:
Done!